rascal2/core/settings.py

```diff
@@ -89,6 +89,7 @@ class Settings(BaseModel, validate_assignment=True, arbitrary_types_allowed=True
 
     log_path: str = Field(default="logs/rascal.log", title="logging", description="Path to Log File")
     log_level: LogLevels = Field(default=LogLevels.Info, title="logging", description="Minimum Log Level")
+    clear_terminal: bool = Field(default=True, title="logging", description="Clear Terminal when Run Starts")
```

Comment:
It think this should be in its own group "terminal" along with other related setting font-size